lib/services/profiles-api-service.js

```diff
@@ -213,6 +215,14 @@ function profileApiServiceFactory(
             });
         })
         .then(function () {
+            if (options.type === 'switch') {
+                return waterline.nodes.updateByIdentifier(
+                    node.id,
+                    { snmpSettings: { host: ip, community: community} }
```

Comment:
@pscharla I do think users will create their own startup-config files, and can use RackHD to deploy them more easily. Even if we do control the startup config settings all the way through, I think it would be better regardless to have this in a task, because then the values (snmp community, etc.) could be synced between the startup config template and the node document via graph options.